component/audioinput/v1/audioinput.pb.gw.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
+	v1_0 "go.viam.com/api/common/v1"
```

Comment:
Not a blocking question, but do we know why this happened?

Reply:
my understanding is that the [formatter is unstable](https://viaminc.slack.com/archives/C01Q5S8V6TB/p1708015325914059?thread_ts=1708011768.501609&channel=C01Q5S8V6TB&message_ts=1708015325.914059)

Reply:
Got it, so this is just how your local code generator behaved. Thanks!